en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -241,12 +247,14 @@ LTE antenna 2 and RC receiver
 
 ### Select Airframe
 
+1. Open QGC and go under vehicle setup.
+
+
 1. Open QGC and go under vehicle setup.
 1. Search for [SYS_AUTO](../advanced_config/parameter_reference.md#SYS_AUTOSTART) in the parameters tab.
-1. Set [SYS_AUTOCONFIG](../advanced_config/parameter_reference.md#SYS_AUTOCONFIG) to "Reset parameters to airframe defaults" and [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) to `13300`.
+1. Set [SYS_AUTOCONFIG](../advanced_config/parameter_reference.md#SYS_AUTOCONFIG) to "Reset parameters to airframe defaults" and [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) to `1230113`.

```

Comment:
13100 would be the generic tiltrortor VTOL, no?

Reply:
My point in https://github.com/PX4/PX4-user_guide/pull/2772/files#r1345065617 is that you shouldn't do any of this. YOu just select the generic airframe by name in the config screen https://docs.px4.io/main/en/config/airframe.html then adjust the geometry.